README.md

```diff
@@ -128,14 +128,20 @@ Config files are in `.json` format:
       "amsgrad": true
     }
   },
-  "loss": "nll_loss",                  // loss
-  "metrics": [
-    "my_metric", "my_metric2"          // list of metrics to evaluate
-  ],                         
+  "loss": {                            // loss function and args
+    "type": "nll_loss",
+    "args": {}
+  },
+  "metrics": {                         // metrics to evaluate and args
+    "my_metric": {},
+    "my_metric2": {
+      "k": 3
+    }
+  },
```

Comment:
The purpose of config file is to make changing between different settings for experiments easier. 
Thus, having arguments of **loss functions** controlled by the config file makes sense, since sometimes they may contain some hyper-parameters. However, that feature does not make any sense for metrics, since all the experiments should have exactly same metric for fair comparison.